gems/policy-parser/Jenkinsfile

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env groovy
```

Comment:
Do we need the Jenkinsfile any more? I see you added testing to the main project's Jenkinsfile.